docs/model/front-end-product-filtering.md

```diff
@@ -41,5 +41,5 @@ Along with filtering the choice will influence the data source for the product l
 You can find more about this topic in [Introduction to Read Model](./introduction-to-read-model.md#read-model-options).
```

Comment:
In topic `Read model option` there is a mention about `services.yml` it should be `services.yaml`, I can't find how to insert suggestion :slightly_frowning_face:

Reply:
👍  I changed all occurrences of services.yml to services.yaml in 98b5118da cc4cccab3 543e798c5 7c7e753f4